newclient/src/modules/address/handlers/AddressTransfers/transfers.graphql

```diff
@@ -1,16 +1,30 @@
 fragment TransferSummary on Transfer {
     transactionHash
+    block
```

Comment:
Can you remove block and status from the fragment

Reply:
removed!